drivers/vfio/pci/nvgrace-gpu/egm.c

```diff
@@ -286,21 +320,21 @@ nvgrace_gpu_fetch_egm_property(struct pci_dev *pdev, u64 *pegmphys,
 	if (ret)
 		return ret;
 
-	if (*pegmlength > type_max(size_t))
+	if (overflows_type(*pegmlength, size_t))
 		return -EOVERFLOW;
 
 	ret = device_property_read_u64(&pdev->dev, "nvidia,egm-base-pa",
 				       pegmphys);
 	if (ret)
 		return ret;
 
-	if (*pegmphys > type_max(phys_addr_t))
+	if (overflows_type(*pegmphys, phys_addr_t))
 		return -EOVERFLOW;
 
 	ret = device_property_read_u64(&pdev->dev, "nvidia,egm-pxm",
 				       pegmpxm);
 
```

Comment:
Also need to check the `ret` value here.

Reply:
I had not modified that as it was part of the original yet-to-be-upstreamed series from Ankit. I'll add a separate commit for it.

FYI, I've gone ahead and reduced the scope of this PR to just the vCMDQ set since that is the priority for QA. I'll resubmit the EGM series in a separate PR next week.